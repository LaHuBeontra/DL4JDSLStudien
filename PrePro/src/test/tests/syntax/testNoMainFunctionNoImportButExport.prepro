const test = 42;
const foo = 13;

test = test * 13;
test = test / foo;

export test, foo;