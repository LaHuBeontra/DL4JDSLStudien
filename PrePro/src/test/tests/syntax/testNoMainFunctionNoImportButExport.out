DefaultPreProDataSet{
dataSetEntries=
[],
dataSet=
null

constants={test=42.00, foo=13.00}

calculated_variables=
test=42.00
foo=13.00
}